latex/express.tex

```diff
@@ -309,14 +304,14 @@ \section{Txantiloiak: EJS}\index{ejs}
 \begin{lstlisting}[language=JavaScript]
 app.get("/", function(req, res) {
     res.render('index', {
-        'izenburua': 'EJS probatzen'    
+        'izenburua': 'Kaixo EJS txantiloi kudeatzailetik'    
     }) 
 });
 \end{lstlisting}
 
 Txantiloia ondo dabilela probatu baino lehen, \hl{public/} karpetatik \hl{index.html} ezabatu egingo dugu. Ez badugu egiten, \textit{express}-ek hor aurkituko du \textit{index} izeneko orri bat eta hori hobetsiko du txantiloiaren aurrean. 
 
-Dena ondo badoa, \hl{http://localhost:3000} webgunea atzitzean, \ref{fig:txantiloia-ejs}. irudian ikusten dugun emaitza lortu beharko genuke.
+Dena ondo badoa, \hl{http://localhost:3000} webgunea atzitzean, \ref{fig:txantiloia1-ejs}. irudian ikusten dugun emaitza lortu beharko genuke.
```

Comment:
[Hau ariketa bat da. Demangun, duda hau daukadala]

Zergatik aldatu duzu label hau? Beharbadaka duplikatuta dago (beste irudiren batek `txantiloia` labela dauka be?